src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -215,61 +124,17 @@ define(function (require, exports, module) {
         },
 
         render: function () {
-            var doc = this.props.document;
-
-            var layerComponents = doc.layers.allVisibleReversed
-                    .filter(function (layer) {
-                        // Do not render descendants of collapsed layers unless
```

Comment:
Another nice catch! I've added this optimization back to the new LayerGroup component.